packages/lesswrong/lib/collections/users/newSchema.ts

```diff
@@ -4189,7 +4189,7 @@ const schema = {
     graphql: {
       outputType: "Boolean",
       canRead: [userOwns, "admins"],
-      canUpdate: ["admins"],
+      canUpdate: [userOwns, "admins"],
```

Comment:
Does the EA forum display the field in a form?  I can only find it in `UsersEditForm`, which I think is the LW account settings form and not used by the EA forum.  Otherwise it's editable by hotkey (opt+shift+s).

Reply:
Oh no it's not in a form, it's just that a user who's a mod and not an admin can't use the hotkeys because they don't have permission to update this field right now